From 5598980edbd93751dc3c006d9d88b1b0e70170cc Mon Sep 17 00:00:00 2001
From: Andy CrossGate Yan <GeForce8800Ultra@gmail.com>
Date: Wed, 27 Oct 2021 14:38:37 +0000
Subject: [PATCH 01/16] Add LineageMonetAccentOverlay

Applies the signature teal of LineageOS (#167C80) when Monet is absent - thanks @kdrag0n

Change-Id: Ib93b2bc68a08dfe12a7c3f503dcc62cce098b459
---
 packages/overlays/Android.mk                  |  1 +
 .../LineageMonetAccentOverlay/Android.bp      | 28 ++++++++
 .../AndroidManifest.xml                       | 22 ++++++
 .../res/values/config.xml                     | 72 +++++++++++++++++++
 4 files changed, 123 insertions(+)
 create mode 100644 packages/overlays/LineageMonetAccentOverlay/Android.bp
 create mode 100644 packages/overlays/LineageMonetAccentOverlay/AndroidManifest.xml
 create mode 100644 packages/overlays/LineageMonetAccentOverlay/res/values/config.xml

diff --git a/packages/overlays/Android.mk b/packages/overlays/Android.mk
index 3a114bc8ec79..15c55d3d588e 100644
--- a/packages/overlays/Android.mk
+++ b/packages/overlays/Android.mk
@@ -26,6 +26,7 @@ LOCAL_REQUIRED_MODULES := \
 	DisplayCutoutEmulationTallOverlay \
 	DisplayCutoutEmulationWaterfallOverlay \
 	FontNotoSerifSourceOverlay \
+	LineageMonetAccentOverlay \
 	NavigationBarMode3ButtonOverlay \
 	NavigationBarModeGesturalOverlay \
 	NavigationBarModeGesturalOverlayNarrowBack \
diff --git a/packages/overlays/LineageMonetAccentOverlay/Android.bp b/packages/overlays/LineageMonetAccentOverlay/Android.bp
new file mode 100644
index 000000000000..e4da9231dc7c
--- /dev/null
+++ b/packages/overlays/LineageMonetAccentOverlay/Android.bp
@@ -0,0 +1,28 @@
+//
+//  Copyright 2021, The Android Open Source Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+package {
+    // See: http://go/android-license-faq
+    // A large-scale-change added 'default_applicable_licenses' to import
+    // all of the 'license_kinds' from "frameworks_base_license"
+    // to get the below license kinds:
+    //   SPDX-license-identifier-Apache-2.0
+    default_applicable_licenses: ["frameworks_base_license"],
+}
+
+runtime_resource_overlay {
+    name: "LineageMonetAccentOverlay",
+    product_specific: true,
+}
diff --git a/packages/overlays/LineageMonetAccentOverlay/AndroidManifest.xml b/packages/overlays/LineageMonetAccentOverlay/AndroidManifest.xml
new file mode 100644
index 000000000000..11062b67bc61
--- /dev/null
+++ b/packages/overlays/LineageMonetAccentOverlay/AndroidManifest.xml
@@ -0,0 +1,22 @@
+<!--
+  ~ Copyright (C) 2021 The Android Open Source Project
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="org.lineageos.overlay.accent.monet"
+        android:versionCode="1"
+        android:versionName="1.0">
+    <overlay android:targetPackage="android" android:priority="0" android:isStatic="true"/>
+</manifest>
diff --git a/packages/overlays/LineageMonetAccentOverlay/res/values/config.xml b/packages/overlays/LineageMonetAccentOverlay/res/values/config.xml
new file mode 100644
index 000000000000..4777409bb587
--- /dev/null
+++ b/packages/overlays/LineageMonetAccentOverlay/res/values/config.xml
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="system_accent1_0">#FFFFFF</color>
+    <color name="system_accent1_10">#EFFFFF</color>
+    <color name="system_accent1_50">#B4FDFF</color>
+    <color name="system_accent1_100">#6EF6FC</color>
+    <color name="system_accent1_200">#4CD9E0</color>
+    <color name="system_accent1_300">#1EBDC4</color>
+    <color name="system_accent1_400">#00A1A7</color>
+    <color name="system_accent1_500">#008388</color>
+    <color name="system_accent1_600">#00696E</color>
+    <color name="system_accent1_700">#004F53</color>
+    <color name="system_accent1_800">#00373A</color>
+    <color name="system_accent1_900">#002022</color>
+    <color name="system_accent1_1000">#000000</color>
+
+    <color name="system_accent2_0">#FFFFFF</color>
+    <color name="system_accent2_10">#EFFFFF</color>
+    <color name="system_accent2_50">#DAF6F7</color>
+    <color name="system_accent2_100">#CCE8E8</color>
+    <color name="system_accent2_200">#B0CCCC</color>
+    <color name="system_accent2_300">#96B0B1</color>
+    <color name="system_accent2_400">#7B9697</color>
+    <color name="system_accent2_500">#617B7C</color>
+    <color name="system_accent2_600">#496364</color>
+    <color name="system_accent2_700">#324B4D</color>
+    <color name="system_accent2_800">#1B3436</color>
+    <color name="system_accent2_900">#051F21</color>
+    <color name="system_accent2_1000">#000000</color>
+
+    <color name="system_accent3_0">#FFFFFF</color>
+    <color name="system_accent3_10">#FDFCFF</color>
+    <color name="system_accent3_50">#EBF1FF</color>
+    <color name="system_accent3_100">#D4E3FF</color>
+    <color name="system_accent3_200">#ACC8F7</color>
+    <color name="system_accent3_300">#91ACDB</color>
+    <color name="system_accent3_400">#7692BE</color>
+    <color name="system_accent3_500">#5C77A2</color>
+    <color name="system_accent3_600">#445F8A</color>
+    <color name="system_accent3_700">#2B4770</color>
+    <color name="system_accent3_800">#113158</color>
+    <color name="system_accent3_900">#001B3D</color>
+    <color name="system_accent3_1000">#000000</color>
+
+    <color name="system_neutral1_0">#FFFFFF</color>
+    <color name="system_neutral1_10">#FAFDFC</color>
+    <color name="system_neutral1_50">#EFF1F1</color>
+    <color name="system_neutral1_100">#E0E3E2</color>
+    <color name="system_neutral1_200">#C4C7C6</color>
+    <color name="system_neutral1_300">#A9ACAB</color>
+    <color name="system_neutral1_400">#8E9191</color>
+    <color name="system_neutral1_500">#737776</color>
+    <color name="system_neutral1_600">#5B5F5F</color>
+    <color name="system_neutral1_700">#444747</color>
+    <color name="system_neutral1_800">#2E3131</color>
+    <color name="system_neutral1_900">#191C1C</color>
+    <color name="system_neutral1_1000">#000000</color>
+
+    <color name="system_neutral2_0">#FFFFFF</color>
+    <color name="system_neutral2_10">#F4FEFE</color>
+    <color name="system_neutral2_50">#E8F3F2</color>
+    <color name="system_neutral2_100">#DAE5E4</color>
+    <color name="system_neutral2_200">#BEC8C9</color>
+    <color name="system_neutral2_300">#A3ADAE</color>
+    <color name="system_neutral2_400">#889393</color>
+    <color name="system_neutral2_500">#6E7878</color>
+    <color name="system_neutral2_600">#566061</color>
+    <color name="system_neutral2_700">#3F4949</color>
+    <color name="system_neutral2_800">#293232</color>
+    <color name="system_neutral2_900">#141D1E</color>
+    <color name="system_neutral2_1000">#000000</color>
+</resources>
-- 
2.25.1

